packages/contractkit/src/utils/phone-number-lookup/matchmaking.ts

```diff
@@ -0,0 +1,99 @@
+import { E164Number } from '@celo/utils/lib/io'
+import crypto from 'crypto'
+import debugFactory from 'debug'
+import {
+  AuthSigner,
+  MatchmakingRequest,
+  MatchmakingResponse,
+  postToPhoneNumPrivacyService,
+  ServiceContext,
+} from './phone-number-lookup'
+
+const debug = debugFactory('kit:registry')
+
+const MATCHMAKING_ENDPOINT = '/getContactMatches'
+// Eventually, the matchmaking process will use blinded numbers same as salt lookups
+// But for now numbers are simply hashed using this static salt
+const SALT = '__celo__'
+const TAG = 'contractkit/utils/phone-number-lookup/matchmaking'
```

Comment:
Do we still need the tag if we use the debugFactory?

Reply:
Good point, I see that debugFactory handles this in the constructor.